app/controllers/pages_controller.rb

```diff
@@ -82,9 +82,10 @@ def terms
   end
 
   def report_abuse
+    billboard_url = params[:billboard]
```

Comment:
I think we want the link to take the moderator to somewhere they can view the problematic billboard. What about something like this?

```suggestion
    billboard_url = admin_display_ad_path(params[:billboard]) if params[:billboard].present?
```